### PR TITLE
Fixed the callback being called after the plugin being unloaded

### PR DESCRIPTION
**Pull request by evandrocoan**

### CODE REVIEW

File: scope_hunter.py [hunk 1]
```diff
@@ -817,6 +817,7 @@ def plugin_loaded():
 
 
 def plugin_unloaded():
-    """Kill the thead."""
+    """Kill the thread."""
 
+    pref_settings.clear_on_change('scopehunter_reload')
```
Comment (facelessuser) [inline scope_hunter.py]: `pref_settings` is not a global variable.

Comment (evandrocoan) [reply scope_hunter.py]: Oops, fixed.

**Comment by facelessuser** [comment]
Cool. E741 error can be ignored as flake8 just updated with this new error.

**Comment by facelessuser** [comment]
Thanks for the pull. I probably need to do this in a couple other plugins.

**Pull request merged**